src/pages/dtrader/index.tsx

```diff
@@ -123,7 +123,8 @@ const Dtrader = () => {
     useEffect(() => {
         setMobile(isBrowser() ? window.screen.width <= size.mobileL : false)
         window.addEventListener('resize', handleResizeWindow)
-    })
+    }, [])
```

Comment:
@mohammad-rework I think you should add `isBrowser` and `handleResizeWindow` as dependencies here instead of an empty array, Would you check on that please 🙏🏻